crates/swc_ecma_minifier/src/compress/util/mod.rs

```diff
@@ -511,18 +511,16 @@ pub(crate) fn eval_as_number(expr_ctx: &ExprCtx, e: &Expr) -> Option<f64> {
                             if args.len() != 2 {
                                 return None;
                             }
-                            let first = eval_as_number(expr_ctx, &args[0].expr)?;
-                            let second = eval_as_number(expr_ctx, &args[1].expr)?;
+                            let base = eval_as_number(expr_ctx, &args[0].expr)?;
+                            let exponent = eval_as_number(expr_ctx, &args[1].expr)?;
 
-                            if second == 0.0 {
```

Comment:
> 2. If exponent is either +0𝔽 or -0𝔽, return 1𝔽.

As step 2 of https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-numeric-types-number-exponentiate, can we simply return 1 instead of trying to evaluate in that condition?

Reply:
It appears that the calculation path has seen minimal alteration.
We generally believe that low-level optimizations, such as those provided by LLVM, typically surpass the performance of manually added branches.